examples/Bremsstrahlung/README.rst

```diff
@@ -1,5 +1,5 @@
-Emission of Bremsstrahlung from Laser-foil interaction
-======================================================
+Bremsstrahlung: Emission of Bremsstrahlung from Laser-foil interaction
```

Comment:
OPTIONAL: 
everywhere you capitalized the title words except here with `interaction`